sources/Debate-frontend/src/app/debate/add-update-debate/add-update-debate.component.ts

```diff
@@ -0,0 +1,94 @@
+import { Component, OnInit, Input, ViewChild } from '@angular/core';
+import { AddUpdateDebateDto } from 'src/app/dto/addUpdateDebate.dto';
+import { ENTER, COMMA } from '@angular/cdk/keycodes';
+import { MatChipInputEvent } from '@angular/material/chips';
+import { DebateService } from 'src/app/services/debate.service';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-add-update-debate',
+  templateUrl: './add-update-debate.component.html',
+  styleUrls: ['./add-update-debate.component.scss']
+})
+export class AddUpdateDebateComponent implements OnInit {
+
+  model: AddUpdateDebateDto;
+  readonly separatorKeysCodes: number[] = [ENTER, COMMA];
+  imageUrl: any;
+  imageData: File;
+  imageInvalid: boolean = false;
+
+  @ViewChild("tagList") tagList;
```

Comment:
```suggestion
  @ViewChild("tagList") tagList: MatChipList;
```
Add tagList field type